src/main/java/duke/ParsedAnswerHandler.java

```diff
@@ -0,0 +1,118 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * This class receives a ParsedAnswer object and execute commands based on information
+ * from the object.
+ */
+
+public class ParsedAnswerHandler {
+    ParsedAnswer pa;
+
+    ParsedAnswerHandler(ParsedAnswer pa) {
+        this.pa = pa;
+    }
+
+    void execute() {
```

Comment:
you can consider having sub-methods to execute the code in each case so the `execute() `method is not too long